widgets/hero/styles/default.less

```diff
@@ -55,6 +56,12 @@
 			max-width: @slide_width;
 			height: @slide_height;
 
+			& when (@vertically_align = true) {
+				display: flex;
+				justify-content: center;
+				flex-direction: column;
```

Comment:
What do you think about moving to alphabetical declarations? _s uses that style and I have been in the themes for a long time. `display, flex-direction and then justify-content` in this case. It's a simple organization method that's easy for everyone to follow.

Reply:
Sounds good. I'll do this now.